sevntu-checks/src/main/resources/com/github/sevntu/checkstyle/checks/coding/messages.properties

```diff
@@ -33,6 +33,7 @@ either.log.or.throw=Either log or throw exception.
 explicit.init=Variable ''{0}'' explicitly initialized to ''{1}'' (default value for its type).
 fall.through=Fall through from previous branch of the switch statement.
 fall.through.last=Fall through from the last branch of the switch statement.
+field.count={0} might be declared only once.
```

Comment:
OK, I think that was meant to be an imperative "may" instead of "might", but "should" is better; done.